sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3358,12 +3358,13 @@ static pi_result ZeDeviceMemAllocHelper(void **ResultPtr, pi_context Context,
   }
   return PI_SUCCESS;
 }
+#endif // 0
 
 // If indirect access tracking is enabled then performs reference counting,
 // otherwise just calls zeMemAllocHost.
 static pi_result ZeHostMemAllocHelper(void **ResultPtr, pi_context Context,
                                       size_t Size) {
```

Comment:
If we agree that you remove enableBufferPooling() switch in this patch (i.e. make pooling enabled by default) then it is safe to remove this helper function.

Reply:
enableBufferPooling